plugins/arangodbendorsements/main.go

```diff
@@ -1003,11 +833,14 @@ func (e *ArangoEndorsementStore) GetMostRecentSwVerOfAnySWComp(args common.Query
 					return nil, fmt.Errorf("query GetMostRecentSwVerOfAnySWComp failed %v", err)
 				}
 				requiredswid, matched = checkMeasMatchInSwIDList(relList, meas)
+				if matched {
```

Comment:
I can't stick a note there, so I'll squat the nearest place:
* L824 the error message looks wrong
* ditto for L833

Reply:
Tidied up the entire chain to clarify with arguments added!